webapp/src/components/team_sidebar/team_sidebar.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import {describe, expect, test, jest} from '@jest/globals';
+import {render} from '@testing-library/react';
+
+import TeamSidebar from './team_sidebar';
+
+const mockTheme = {
+    sidebarBg: '#ffffff',
+    sidebarText: '#333333',
+    sidebarUnreadText: '#ff0000',
+    sidebarTextHoverBg: '#eeeeee',
+    sidebarTextActiveBorder: '#007bff',
+    sidebarTextActiveColor: '#007bff',
+    sidebarHeaderBg: '#f8f9fa',
+    sidebarHeaderTextColor: '#495057',
+    onlineIndicator: '#28a745',
+    awayIndicator: '#ffc107',
+    dndIndicator: '#dc3545',
+    mentionBg: '#ffeb3b',
+    mentionBj: '#ffeb3b',
+    mentionColor: '#333333',
+    centerChannelBg: '#f8f9fa',
+    centerChannelColor: '#333333',
+    newMessageSeparator: '#007bff',
+    linkColor: '#007bff',
+    buttonBg: '#007bff',
+    buttonColor: '#ffffff',
+    errorTextColor: '#dc3545',
+    mentionHighlightBg: '#ffeb3b',
+    mentionHighlightLink: '#007bff',
+    codeTheme: 'solarized-dark',
+};
+
+jest.mock('../sidebar_buttons', () => ({
+    __esModule: true,
+    default: () => <div data-testid='mocked-sidebar-buttons'/>,
+}));
+
+describe('TeamSidebar', () => {
+    test('renders nothing when show is false', () => {
```

Comment:
I think it's always worth adding a snapshot test if possible when introducing tests for a component. Are we able to use `toMatchSnapshot` from jest here with the `@testing-library/react` library?
https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest

Reply:
@mickmister , I have rewritten the tests to make use of jest snapshots.